src/components/app/Layout.jsx

```diff
@@ -44,7 +44,7 @@ const Layout = () => {
       <main id="content" className="fill-vertical-space">
         <Outlet />
       </main>
-      <SiteFooter />
+      <FooterSlot />
```

Comment:
[inform] Misc. improvement to support a plugin-enabled footer; basically a no-op though.